starter-code/server.js

```diff
@@ -66,18 +83,28 @@ app.put('/articles/:id', function(request, response) {
   // an author_id property, so we can reference it from the request.body.
   // TODO: Add the required values from the request as data for the SQL query to interpolate
   client.query(
-    ``,
-    []
+    `UPDATE authors(author, "authorUrl")
+     VALUES ($1, $2)
+     WHERE author = $1`,
+    [request.body.author,
+      request.body.authorUrl]
   )
```

Comment:
So with this, you aren't actually setting any of these properties, the way you have written this query string, which is what you want to accomplish with a put. It would look more like `UPDATE authors  SET property=$1, property=$2 ... WHERE property = $3`, [ ].